download-qemu-static.sh

```diff
@@ -1,33 +1,43 @@
 #!/bin/bash -ex
 
-set -xe
+set -eux
+
 if [ "$(uname -m)" == "x86_64" ]; then
     docker run --rm --privileged multiarch/qemu-user-static:register --reset
 fi
 
-export QEMU_STATIC_VERSION=v7.2.0-1
-qemu_aarch64_sha256=dce64b2dc6b005485c7aa735a7ea39cb0006bf7e5badc28b324b2cd0c73d883f
-qemu_arm_sha256=9f07762a3cd0f8a199cb5471a92402a4765f8e2fcb7fe91a87ee75da9616a806
-qemu_ppc64le_sha256=a8855b9a9cdefbe2163d9f7851fb71c77207d816451237caed616eb9b03229ac
-qemu_s390x_sha256=a438ab2f7c2e0f0ffe63992bccedaf60d789cfb1849e035c0764bda7d9e73a9a
-
-set +e
-rm qemu-*-static
-set -e
-
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-aarch64-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-arm-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-ppc64le-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-s390x-static
+rm -f qemu-*-static
 
-sha256sum qemu-*-static
+set -- aarch64 arm ppc64le s390x
```

Comment:
Could you use more standard bash constructs to set and loop over `arch` here? If we're using `arch` below, we should be setting that (or a collection of it) somewhere, otherwise the code becomes unnecessarily cryptic (and fragile) IMO.

Reply:
> more standard bash constructs

I mean, that's standard POSIX :laughing:.
But yes, let's simplify this.
Since this now uses `dpkg-deb --extract` and only loops `arch` ones instead of having `bsdtar` with the file list additionally, we can remove the `set`.